yarn-project/simulator/src/client/simulator.test.ts

```diff
@@ -1,90 +0,0 @@
-import { CompleteAddress, Note } from '@aztec/circuit-types';
```

Comment:
This was just testing calling the compute note hash deleted fn, so I got rid of it.